.do/db-cluster/databases.tf

```diff
@@ -0,0 +1,34 @@
+resource "digitalocean_database_cluster" "postgres" {
+  for_each = local.environments
+  name = "codacy-doks-cluster-postgres${var.postgres_version}-${each.key}"
```

Comment:
I am not sure if we should put the postgres.version in the name.
Doesn't the DB get replaced when we set this to "11"? Instead of just updating "in place"?